internal/bminventory/inventory.go

```diff
@@ -4225,7 +4214,6 @@ func (b *bareMetalInventory) DisableHost(ctx context.Context, params installer.D
 
 	txSuccess := false
 	tx := b.db.Begin()
-	tx = transaction.AddForUpdateQueryOption(tx)
```

Comment:
What is the downside of doing it that way?
I think that other developers will forget to put this no?

Reply:
I think we need to modify the common query functions that they will support this option.
The "for update" clause has to issued on the initial query in a transaction.  After that, the relevant 
records are locked until the end of the transaction

Reply:
But this will be probably part of a different task

Reply:
Yes it is out of scope for this PR, please open another ticket for that.

Reply:
https://issues.redhat.com/browse/MGMT-8252